azurelinuxagent/common/protocol/wire.py

```diff
@@ -692,6 +695,23 @@ def update_certs(self, goal_state):
         self.save_cache(local_file, xml_text)
         self.certs = Certificates(self, xml_text)
 
+    def update_remote_access_conf(self, goal_state):
+        if goal_state.remote_access_uri is None:
+            raise ProtocolError("RemoteAccess uri is empty")
+        local_file = os.path.join(conf.get_lib_dir(), REMOTE_ACCESS_FILE_NAME)
+        xml_text = self.fetch_config(goal_state.remote_access_uri, 
+                                     self.get_header_for_cert())
+        self.save_cache(local_file, xml_text)
+        self.remote_access = RemoteAccess(xml_text)
+
+    # def update_remote_access_conf(self, goal_state):
```

Comment:
remove commented code

Reply:
removed

---
In reply to: [192192637](https://github.com/Azure/WALinuxAgent/pull/1171#discussion_r192192637) [](ancestors = 192192637)